ui/admin/app/styles/app.scss

```diff
@@ -613,6 +613,10 @@
       flex-direction: column;
       align-items: center;
       text-align: center;
+
+      .hds-icon-tile--color-hcp {
+        background-color: var(--subtle);
+      }
```

Comment:
I would prefer not to provide this override to the default HDS styles, if possible.  Over time these accumulate.

Reply:
Removed.